docs/sources/setup-grafana/set-up-grafana-monitoring.md

```diff
@@ -81,8 +81,10 @@ These instructions assume you have already added Prometheus as a data source in
    ```
 
 1. Restart Prometheus. Your new job should appear on the Targets tab.
-1. In Grafana, hover your mouse over the **Configuration** (gear) icon on the left sidebar and then click **Data Sources**.
+1. In Grafana, click **Connections** in the left-side menu.
+1. Under your connections, click **Data Sources**.
 1. Select the **Prometheus** data source.
+1. Under the name of your data source, click **Dashboards**.
 1. On the Dashboards tab, **Import** the Grafana metrics dashboard. All scraped Grafana metrics are available in the dashboard.
```

Comment:
Does this mean I'm clicking a button labeled Import?
I'm not familiar at all with this UI, but I'd make this last added step clearer about the action the user needs to take.